packages/libraries/router/src/logger.rs

```diff
@@ -0,0 +1,57 @@
+use once_cell::sync::Lazy;
+use std::env;
+
+pub struct Logger {}
+
+static CURRENT_LOG_LEVEL: Lazy<String> =
+    Lazy::new(|| env::var("RUST_LOG").unwrap_or_else(|_| "info".to_string()));
+
+const LOG_LEVEL_SCORES: [(LogLevel, u8); 5] = [
+    ("debug", 5),
+    ("info", 4),
+    ("warn", 3),
+    ("error", 2),
+    ("trace", 1),
+];
+
+type LogLevel = &'static str;
+
+impl Logger {
+    fn find_log_level_score(log_level: &str) -> u8 {
+        LOG_LEVEL_SCORES
+            .iter()
+            .find(|(l, _)| *l == log_level)
+            .map(|(_, score)| *score)
+            .expect("Invalid log level")
+    }
```

Comment:
I guess this could also be as you've mentioned before, just an Array with those items and we depend on the index instead of the tuples with scores